bbn-1/network-artifacts/upgrades/v2/README.md

```diff
@@ -2,31 +2,31 @@
 
 ## Upgrade overview
 
-- **Upgrade version**: `v2.0.0`
+- **Upgrade version**: `v2.1.0`
```

Comment:
If someone is trying to sync from scratch and use v2.1 since the v2 upgrade would it get an apphash? it might need some testing